lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -1497,6 +1497,7 @@ struct FIRRTLLowering : public FIRRTLVisitor<FIRRTLLowering, LogicalResult> {
   LogicalResult visitDecl(VerbatimWireOp op);
 
   // Unary Ops.
+  LogicalResult visitSizeOfIntrinsiicOp(SizeOfIntrinsicOp op);
```

Comment:
I've been meaning to fix this, but I don't think `SizeOfIntrinsicOp` made it to the Visitor thing, FWIW.

But I don't think this belongs here as it's not defined (and has a typo).